### PR TITLE
dotCMS/core#25028 [UI] Add a Go to Report Experiment button in a Edit Page if the page has an RUNNING Experiment 

### LINKED ISSUE #25028: [UI] Add a Go to Report Experiment button in a Edit Page if the page has an `RUNNING` Experiment
### Parent Issue

_No response_

### User Story

As a user, I want to see a button in the Edit Page interface that allows me to go directly to the reports page if the page has a running experiment.

### Acceptance Criteria

- When a user is editing a Page and the Page has a running experiment, a button should be displayed.
- Clicking the button should navigate the user to the reports page for that experiment.
- The button should only be shown if there is a RUNNING experiment on the Page.
- If there are no running experiments on the Page, the button should not be displayed.

### Proposed Objective

User Experience

### Proposed Priority

Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

image

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by hmoreras**
### Proposed Changes
* If an experiment is running under a page, show indicator tag that works as a link to the experiment results. 

### screenshots
image

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/services/dot-experiments.service.ts [hunk 1]
```diff
@@ -47,6 +47,18 @@ export class DotExperimentsService {
             .pipe(pluck('entity'));
     }
 
+    /**
+     * Get an array of experiments of a pageId that are running
+     * @param {string} pageId
+     * @returns Observable<DotExperiment[]>
+     * @memberof DotExperimentsService
+     */
+    getRunning(pageId: string): Observable<DotExperiment[]> {
```
Comment (oidacra) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/services/dot-experiments.service.ts:56]: change to getByStatus and send pageId and a ExperimentStatus to do a little bit more generic.

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.html [hunk 2]
```diff
@@ -78,6 +78,31 @@ <h2>{{ variant.experimentName }}</h2>
 
 <ng-template #defaultHeaderRightTpl>
     <dot-global-message data-testId="globalMessage" right></dot-global-message>
+    <p-tag
```
Comment (oidacra) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.html:81]: Add a <ng-container *ngIf.... to handle only one IF, the tag and the button wont be showed if dont have any experiment.

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts [hunk 3]
```diff
@@ -634,4 +639,16 @@ browse from the page internal links
...
+    private getRunningExperiment(): void {
+        this.pageState$.pipe(take(1)).subscribe((content) => {
+            this.runningExperiment$ = this.dotExperimentsService
+                .getByStatus(content.page.identifier, DotExperimentStatusList.RUNNING)
+                .pipe(
+                    map((experiments) => {
+                        return experiments.length ? experiments[0] : null;
+                    })
+                );
+        });
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts:652]:
```suggestion
this.runningExperiment$ = this.pageState$.pipe(
    take(1),
    switchMap((content) =>
        this.dotExperimentsService.getByStatus(content.page.identifier, DotExperimentStatusList.RUNNING)
    ),
    map((experiments) => experiments.length ? experiments[0] : null)
);
```

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.html (hunk 2 above)
Comment (hmoreras) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.html:82]: just moved to one element not needed anymore.

**Review by oidacra — APPROVED** [review]

**Review by oidacra — APPROVED** [review]

**Review by zJaaal — APPROVED** [review]

**Review by fmontes — APPROVED** [review]

**Review by rjvelazco — APPROVED** [review]

**Pull request merged**